python/langsmith/env/__init__.py

```diff
@@ -27,4 +27,5 @@
     "get_runtime_environment",
     "get_system_metrics",
     "get_git_info",
+    "exec_git",
```

Comment:
Let's not export this on the `env` level (don't want to update our public API). We can still use with other internal files though